### ISSUE TITLE
Several issues with El Capitan installation

### ISSUE THREAD

**Issue by deankhoward** [RanbirAulakh/Asus-Q501LA-Hackintosh#12]
going through the tutorial, these are the issues i faced.

1) I could not create 2 partitions on a USB using MBR for the boot options using El Capitan Disk Utilitiy, Neither terminal nor Disk Utility would allow me. tried GUID anyways
2)  I was eventually able to boot to mac, however every time i tried to install it, it would claim that the Install media was corrupted most likely during download. I tried the setdate technique, but my solution was just to create a seperate boot installation media and i used the first usb (as used in the tutorial) to boot with clover, and install off the second USB with only the media. Once installed installed Clover and ticked all that were suggested. 
3) the tutorial says to extract the EFI downloaded from github to my EFI partition. Does that mean all the folders in the EFI partition should be overwritten by  the EFI from Github? or does that mean the EFI partition has all its folders PLUS a folder titled EFI with its own subfolders? i copied all the sub folders and overwrote the folders in the EFI partition.
4) My final issue is i couldnt find the FAKEPCIID in the download. it gave me a bunch of stuff but no kext. am i missing something? I dragged all the kexts in the EFI partition to KExt utility and they installed. I
5) i still get graphic errors upon every boot (requiring the close lid technique). I feel this may be solved with the FakePcIID. 
6) it only reports the 4gb of ram soldered to my motherboard, not the additional 4gb Stick installed. considering what i was planning on doing with this, this makes the computer pretty much pointless. 

Thank you in advance for taking a look and helping me out.

**Comment by RanbirAulakh** [comment]
@deankhoward I do not have access to the Laptop anymore, won't be able to use it until Summer (last week of May).

1) Try format the whole USB as Mac Journaled before doing paritions?
2) It shouldn't happen, not sure why.
3) Take EFI folder in EFI.zip file and put them in CLOVER
4) Go to [FakePCIID link](https://bitbucket.org/RehabMan/os-x-fake-pci-id/downloads) and download the first one, open zip file > release and copy FacePCIID.kext and FakePCIID_Intel_HD_Graphics.kext and drag them into Kext Utility. 
5) Try above ^
6) 4gb soldered to motherboard? What laptop do you have?

Maybe @shamesung and @gvkt will help?

**Comment by deankhoward** [comment]
![20160423_143310](https://cloud.githubusercontent.com/assets/14366903/14764101/91bfc696-0960-11e6-8490-9d35866cd50f.jpg)
@RanbirAulakh  Here you can see the 4gb of ram soldered (left) and the 4gb stick (right). you can also see where it states the model number. I think Asus released two versions of the Q501LA. one with 6gb of ram (2 soldered +4gb) and one with 8gb of ram (4 soldered +4gb). I installed the fakePciid and now i only have a slight graphic hiccup when booting, but it clears up at the login screen. thank you for the quick reply. Do you think buying mac specific laptop modules would help? the ram shows up fine in Windows.

**Comment by RanbirAulakh** [comment]
@deankhoward I do not think that will help that much, that would be waste of $, I wouldn't do it. If you can't get that to work, I recommend going over to [tonymacx86](http://www.tonymacx86.com/el-capitan-laptop-support/) and make a post about it, they will help you out how to get those other 4gb to work. For me, if I can remember correctly, it full uses 6gb ram.

**Comment by deankhoward** [comment]
@RanbirAulakh Once again, i appreciate the fast response. my bios definitely detects the total 8gb of ram. Ill try tonymacx86, but for the sake of clarity, im not sure how to describe my problem. So far i've gotten a semi-stable hackintosh (tbh i have very little experience and im not sure what im missing) with a few hiccups with my trackpad and half the ram not detected.

**Comment by shamesung** [comment]
@deankhoward Fix for ram issue http://www.insanelymac.com/forum/topic/309226-os-x-is-only-recognizing-half-of-my-ram/page-1#entry2186780

**Comment by shamesung** [comment]
You need clover configurator

**Comment by shamesung** [comment]
You likely have Q502-LA like I do
First with About Mac note down the type, manufacturer, speed, size and serial number of the RAM that is already detected by your laptop
Then In Clover Configurator, add (click the plus sign):
SLOT  SIZE FREQUENCY VENDOR PART SERIAL TYPE
1        THE INFO YOU NOTED DOWN
2  SAME SIZE, SAME FREQUENCY, ASUS, UNKNOWN, ENTER 0000's as long as the serial number above, SAME TYPE

_In our case note the size for each module is 4096 - so you should enter 4096 for slot 1 and slot 2_
![screenshot 2015-11-08 04 53 13](https://cloud.githubusercontent.com/assets/6269751/14764977/d35c918e-0991-11e6-937d-92f9f80e5e1c.png)

**Comment by shamesung** [comment]
audio should be relatively fine, graphics has some bugs but its manageable,i recommend you install the latest AppleHDA, CodecCommander, FakePCIID_Intel_HD_Graphics from here (https://bitbucket.org/RehabMan/os-x-fake-pci-id/downloads), trackpad can be fixed by installing the latest version of this kext and setting scrolling speed to maximum value http://forum.osxlatitude.com/index.php?/topic/1948-elan-focaltech-and-synaptics-smart-touchpad-driver-mac-os-x/ ,  hdmi works but i can't figure out the audio, wifi needs a new card the intel ones are not recognized by os x

**Comment by shamesung** [comment]
See here if you have any other issues, https://github.com/shamesung/Asus-Q501LA-Q502LA-Hackintosh/wiki/4.-FAQ
